examples/generic-st77xx-with-xpt2046.py

```diff
@@ -0,0 +1,46 @@
+import sys
```

Comment:
> Perhaps moving them to the driver itself would be better? So that there is e.g. `driver/generic/st77xx.py` and `driver/generic/st77xx-test.py`

Yes I think that would be clearer.  The "examples" directory is really for LVGL examples and not for driver test code.  
Also, you won't need to exclude each of these tests on `tests/run.sh`, which by default tests every example under "examples"

Reply:
Done.